auth_supplier/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+Auth Supplier
+=============
+
+This module was written to extends the functionality of auth signup
+and allows the user to create an account as a supplier.
+
+Configuration
+=============
+
+* To enable users to create accounts:
+
+* Go to *Settings > Configuration > General settings*.
+* Enable *Allow external users to sign up*.
+* Enable *Activate the customer portal*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Log out.
+* In home page, press *Sign in*.
+* Press *Sign up*.
+* Select *Supplier* in account type.
```

Comment:
I cannot find this option, event I follow configuration steps

![29-10-2015 19-41-18](https://cloud.githubusercontent.com/assets/8736623/10828514/1643c2e0-7e75-11e5-9b1e-68a8a15772d6.png)